#13797603: Модальное окно бронирования и подтверждения заказа

Lakatosska opened the pull request:
1. Task: Модальное окно бронирования и подтверждения заказа

3. Screenshot:
![image](https://github.com/agricultural-machinery-rental/client/assets/89295707/a936b058-2ffe-4d53-9368-c7f0ab8dd801)
![image](https://github.com/agricultural-machinery-rental/client/assets/89295707/c1dbadab-6c9f-41ce-9d8f-53436cefa71e)


4. Was done:
   - [x] Модальное окно бронирования
   - [x] Модальное окно успешного бронирования (подтверждения заказа)

Lakatosska committed:
feat: add booking modal

Lakatosska committed:
feat: add booking modal

Lakatosska committed:
feat: add booking success modal

Lakatosska committed:
feat: add booking success modal

Lakatosska committed:
fix: styles

Lakatosska committed:
fix: styles

djpont commented on src/widgets/user/Booking/ui.tsx:
```diff
@@ -0,0 +1,44 @@
+
+  const handleBookingConfirm = () => {
+    // TODO добавить submit формы
+    closeModal();
+    openModal(modalWindows.bookingSuccess);
```
Можно не закрывать, а сразу открыть новую. Она всё равно одна и та же.

Lakatosska replied:
ну думаю лучше закрыть, а то будешь что-нибудь потом по стилям перекрывать или еще что-то всплывет. в общем, момент не принципиальный, да  )

Catalinadevolg commented on src/shared/ui/Textarea/styled.ts:
```diff
@@ -22,6 +23,8 @@ export const StTextarea = styled.textarea<TTextareaStyledComponent>`
   height: 112px;
   width: 100%;
+  font-size: 0.6rem;
+  }
```
Скобочка лишняя.
Добавь, пожалуйста, `line-height: 1rem;` сюда, а то не хватает

Catalinadevolg commented on src/widgets/user/Booking/ui.tsx:
```diff
@@ -0,0 +1,44 @@
+        label='Забронировать'
+        $designType='primary'
+        // TODO заменить console.log
+        onClick={() => console.log('Отправить данные бронирования')}
```
Для отправки данных не нужно передавать функцию в `onClick` кнопки. Нужно передать функцию в `handleFormSubmit` компонента формы.

Catalinadevolg commented on src/widgets/user/Booking/config.ts:
```diff
@@ -0,0 +1,18 @@
+  {
+    kindOfField: 'textarea',
+    name: FormFieldNames.message,
+    required: true,
```
В чате решается вопрос по комментарию. Вероятно сделаем его не обязательным к заполнению.

Catalinadevolg commented on src/widgets/user/Booking/config.ts:
```diff
@@ -0,0 +1,18 @@
+    kindOfField: 'textarea',
+    name: FormFieldNames.message,
+    required: true,
+    type: 'text',
```
У textarea нет атрибута `type`. Убери, пожалуйста.

Catalinadevolg commented on src/shared/ui/DateRange/styled.ts:
```diff
@@ -5,6 +5,7 @@ import { theme } from '@/shared/styles/theme';
 export const StDateRangeWrapper = styled.div`
```
Можешь, пожалуйста, у инпута шрифты поправить под фигму? Сейчас размер выбранной даты огромный.

Catalinadevolg commented on src/shared/ui/DateRange/styled.ts:
```diff
@@ -5,6 +5,7 @@ import { theme } from '@/shared/styles/theme';
 export const StDateRangeWrapper = styled.div`
   position: relative;
   display: inline-block;
+  z-index: 1;
```
Вынеси, пжл, в константы с z-index

tltelf committed:
#12898034: layout product-card (#57)

1. Task: Изменена верстка и стили в соответствии с макетом

2. Was done:
   - [x] Изменена верстка и стили

Lakatosska committed:
fix: date range styles

Lakatosska committed:
fix: styles

Lakatosska committed:
fix: styles
